backend/index.js

```diff
@@ -2,12 +2,15 @@ const express = require('express')
 const cors = require('cors')
 
 const router = require('./api/router')
+const runCrons = require('./cron')
 
 const app = express()
 
 app.use('/api/v1', cors(), router)
 
-const port = 8001
+runCrons()
+
+const port = 8002
```

Comment:
why the port change? :confused:

Reply:
Anyway, you can already change to `process.env.PORT || 3000`

Reply:
I was experimenting with something and completely missed it! sorry!